src/app/components/forms/select-box/terra-select-box.component.ts

```diff
@@ -394,6 +396,13 @@ export class TerraSelectBoxComponent implements OnInit, OnChanges
         }
     }
 
+    private selectFallbackValue():void
+    {
+        this.selectedValue = this.inputListBoxValues[0];
```

Comment:
was passiert in dem Fall, wenn keine values vorhanden sind? sollte die Liste zumindest leer initialisiert werden?